incident_post_mortems/README.md

```diff
@@ -0,0 +1,42 @@
+# Incident post mortems
+Though we fully expect to write dependable applications, every project will
+experience service disruptions and other significant failings. In all cases,
+we want to learn from our mistakes both within our projects and more broadly.
+Post mortems, which detail the events and how they were resolved, are an
+excellent mechanism for sharing this information.
+
+Note that this document won't discuss what to do during a security incident,
+if cloud.gov is having issues, what to report to your client, etc. For those,
+see:
+* https://github.com/18F/security-incidents
+* https://cloudgov.statuspage.io/
+* cloud-gov-support@gsa.gov
+* Slack: #infrastructure, #incident-response, #cloud-gov-support
+
+## Key components
```

Comment:
This might be a good place to work in the Mark Imbriaco formula:

> 1. Apologize for what happened.
> 2. Demonstrate you understand what happened.
> 3. Explain what you will do to reduce the likelihood of it happening again.

(see https://www.digitalocean.com/company/blog/inside-digitalocean-mark-imbriaco/)